user-manager/src/main/java/com/qpay/usermanager/model/entity/customer/CustomerEntity.java

```diff
@@ -0,0 +1,33 @@
+package com.qpay.usermanager.model.entity.customer;
+
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import jakarta.persistence.Entity;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor(access = AccessLevel.PROTECTED)
```

Comment:
why do you need access level?

Reply:
i thought, it would be good if constructors have scope only in package or in inheritance classes, because creating instances of this class have to be done by @Builder